cmd/username_populator/main.go

```diff
@@ -0,0 +1,88 @@
+package main
+
+import (
+	"context"
+	"database/sql"
+	"log"
+	"os"
+
+	"github.com/Molnes/Nyhetsjeger/internal/database"
+	"github.com/joho/godotenv"
+)
+
+func main() {
+	err := godotenv.Load()
+	if err != nil {
+		log.Fatal("DB Populator: Error loading .env")
```

Comment:
should not be a fatal error, since the environment isn't necessarily in the .env

Reply:
fixed